docker_explorer/de.py

```diff
@@ -179,24 +216,50 @@ def Mount(self, container_id, mountpoint):
       container_id (str): the ID of the container.
       mountpoint (str): the path to the destination mount point.
     """
-    if self.storage_object is None:
-      self.DetectStorage()
-    self.storage_object.Mount(container_id, mountpoint)
+    container_object = self.GetContainer(container_id)
+    self.storage_object.Mount(container_object, mountpoint)
+
+  def GetContainersString(self, only_running=False):
+    """Returns a string describing the running containers.
+
+    Args:
+      only_running (bool): Whether we display only running Containers.
+    Returns:
```

Comment:
Add a newline (starting to wonder if that's in fact what the style guide specifies)

Reply:
https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments
There are indeed newlines there